src/applications/personalization/va-profile/tests/e2e/routes.json

```diff
@@ -0,0 +1,180 @@
+[
+  {
+    "path": "/v0/address/countries",
+    "verb": "get",
+    "value": {
+      "data": {
+        "attributes": {
+          "countries": [
+            "USA"
+          ]
+        }
+      }
+    }
+  },
+  {
+    "path": "/v0/address/states",
+    "verb": "get",
+    "value": {
+      "data": {
+        "attributes": {
+          "states": [
+            "Alabama",
+            "Kentucky"
+          ]
+        }
+      }
+    }
+  },
+  {
+    "path": "/v0/profile/full_name",
+    "verb": "get",
+    "value": {
+      "data": {
+        "attributes": {
+          "first": "First",
+          "middle": "Middle",
+          "last": "Last"
+        }
+      }
+    }
+  },
+  {
+    "path": "/v0/profile/email",
+    "verb": "get",
+    "value": {
+      "data": {
+        "attributes": {
+          "email": "old_email@va.gov"
+        }
+      }
+    }
+  },
+  {
```

Comment:
After moving to Vet360, we'll have to edit the mock for the User data and remove these.